test/discharge-wizard/00-required.e2e.spec.js

```diff
@@ -24,16 +24,22 @@ module.exports = E2eHelpers.createE2eTest(
       .setValue('input[name="1_branchOfService"]', '1')
       .click('input[name="1_branchOfService"]')
       .selectDropdown('2_dischargeYear', '2016')
+      .waitForElementPresent('input[name="4_reason"]', Timeouts.normal)
```

Comment:
I'm generally okay with adding these kinds of waits to the e2e tests.

@jbalboni I forget how this one works. Would it make more sense for it to be `.waitForElementVisible`?

Reply:
For most of these the reason is because I tried both and it worked when I used Present. I think the specific problem is that sometimes these elements appear off screen in in the selenium window. Confusingly this only sometimes seems to change the results of the `.waitForElementVisibleCall`.